### DISCUSSION TITLE
Boat/tank - Poshold cannot be enabled

### DISCUSSION THREAD

**Discussion by j-lang** [iNavFlight/inav#9684] [Support] [answered]
Built a boat (tank) configuration with two motors similar to a tank mode according to the blog article of Pawel: https://quadmeup.com/inav-for-tracked-rovers/ 
A Speedybee 405 Wing with GPS and bidirectional ESCs are used. Firmware: 7.0.0,  INAV Configurator: 7.0.1
After finishing configuration and tests the Poshold (Loiter) Mode should also be activated, but the Poshold mode is not listed in mode-settings. The Poshold mode requires GPS and baro and both are available and active. 

Are there some other prerequisites get the poshold mode active and usable?

**Comment by DzikuVx** [comment, marked as answer]
PosHold is not supported on those platforms yet.
In case of ground vehicles it's not requied and ground vehicle will not go anywhere with stopped motors.
For boats it's not yet implemented
